tests/integration/it_reservations_test.py

```diff
@@ -3,30 +3,25 @@
 from flask import url_for
 
 import pytest
-
 from timeless.restaurants.models import Reservation, ReservationStatus
 
+def test_create_reservation(client, db_session):
+    reservation = Reservation.query.get(100)
+    if reservation:
+        db_session.delete(reservation)
+        db_session.commit()
 
-@pytest.mark.skip
-def test_create_reservation(client):
-    """
-    @todo #235:30min Find out why form validation is failing for the
-     create_reservation test. Fields that are reported to be invalid
-     are start_time, end_time and status. Remove skip annotation
-     once form validation is fixed.
-    """
-    comment = "my comment"
     response = client.post(url_for("reservations.create"), data={
-        "start_time": datetime.utcnow(),
-        "end_time": datetime.utcnow(),
-        "comment": comment,
+        "id": 100,
+        "start_time": datetime(2019, 2, 20, 21, 30),
+        "end_time": datetime(2019, 2, 20, 22, 30),
+        "customer_id": 1,
         "num_of_persons": 2,
-        "status": ReservationStatus.confirmed
+        "comment": "my comment",
+        "status": ReservationStatus.confirmed.name
     })
     assert response.location.endswith(url_for("reservations.list_reservations"))
     assert Reservation.query.count() == 1
```

Comment:
@marceloamadeu we need to add assert to check the inserted data is okay

Reply:
@ammaratef45 Added statement to check inserted data.